contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -246,6 +246,7 @@ modify_date timestamptz NOT NULL,
 default_database_name SYS.NVARCHAR(128) NOT NULL,
 default_language_name SYS.NVARCHAR(128) NOT NULL,
 properties JSONB,
+orig_loginname SYS.NVARCHAR(128) NOT NULL,
```

Comment:
There are two cases,
1. Newly created login with this change should show value for orig_loginname matching with what user has provided
2. In case of upgrade, It should be NULL for whatever rows which are already there. 

So, 
1. default value should be NULL and need to remove NOT NULL check
2. Update upgrade script accordingly 
3.  Add upgrade test to test the behaviour.